src/components/VTranslationStatusBanner.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <VNotificationBanner
-    :id="bannerKey"
+    id="translation"
```

Comment:
This prop needs to be unique between all instances as `VNotificationBanner` uses it to track which banners have been dismissed. With this change all translation banners have the same id here and if you dismiss one, all the others will also be treated as dismissed. You can try this by clearing your local storage, visiting `/pt` and dismissing the banner then visiting `/ru` and see that the banner will already be dismissed.

Is there any reason we can't use `data-testid` instead?

Reply:
I saw why I wasn't using `data-testid`: it's because the `VNotificationBanner` uses the `id` prop to create its own `data-testid` that is set as `banner-${id}`, so it becomes something like `data-testid="banner-translation-ar"` :(

Reply:
Buuut, I can override the `VNotificationBanner`'s `data-testid` by providing my own `data-testid` from `VTranslationStatusBanner`, so it works well!